packages/schema/src/index.ts

```diff
@@ -4,6 +4,10 @@ export * from './pagination/pagination'
 export * from './pagination/pagination.types'
 
 export * from './api-key'
+
+export * from './auth/auth'
+export * from './auth/auth.types'
```

Comment:
You shouldn't export the types from here. Types should be exported in the `index.types.d.ts` file